sklearn/metrics/_plot/uncertainty.py

```diff
@@ -0,0 +1,262 @@
+"""
+TODO: Documentation
+
+AISTAT 2023 `Sampling uncertainties on the Precision-Recall curve`
```

Comment:
Correct title of paper is: `Pointwise sampling uncertainties on the Precision-Recall curve`
with authors: `R.E.Q. Urlus, M.A. Baak, S. Collot, I. Fridman Rojas`